app/__init__.py

```diff
@@ -197,6 +199,24 @@ def get_spots():
     spot = Spot.query.filter_by(id=beach_id).first()
     spot_data = spot.__dict__
     spot_data.pop('_sa_instance_state', None)
+    ###################################
+    reviews = db.session.query(db.func.count(Review.rating), Review.rating).filter_by(beach_id=beach_id).group_by(Review.rating).all()
+    count = Review.query.filter_by(beach_id=beach_id).count()
+    average = db.session.query(db.func.avg(Review.rating)).filter_by(beach_id=beach_id).all()
```

Comment:
database calls are expense (aka slow), so we shouldn't use them for calculations if we have the data we could use to calculate it already fetched. so i would either remove this, since we don't need it, or just calculate it by averaging together the reviews that we just fetched. imo we should remove it for now completely

Reply:
I took out the count and commented out the average. I made the front end use the previously available values for total reviews and average review. I made it into a helper function so that it could be used in both. The standalone as well as in the spots/get[beach_id]